src/raygui.h

```diff
@@ -1612,7 +1612,7 @@ int GuiTabBar(Rectangle bounds, const char **text, int count, int *active)
     #define RAYGUI_TABBAR_ITEM_WIDTH    160
 
     int result = -1;
-    GuiState state = guiState;
+//    GuiState state = guiState;
```

Comment:
i prefer to delete unused code, but i saw a pattern of commenting them out instead (such as on line #1480, #3409, etc)

Reply:
By raylib/raygui conventions, comments for unused variable are always in fornt of the variable, with no spaces.